data/tlg0081/tlg013/tlg0081.tlg013.1st1K-grc1.xml

```diff
@@ -0,0 +1,1192 @@
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Serina Hu</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Peter Hulse</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Mert İnan</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Daria Kondakova</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Felipe Antonio Pinilla Torres</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Ruth Elizabeth Slater</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Charlotte Houghton</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digial editor</resp>
+      </respStmt>
+    </titleStmt>
+    <publicationStmt>
+      <publisher>Open Greek and Latin</publisher>
+      <idno type="filename">tlg0081.tlg013.1st1K-grc1.xml</idno>
+      <pubPlace />
+      <availability>
+        <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available
+        under a Creative Commons Attribution-ShareAlike 4.0 International
+        License</licence>
+      </availability>
+      <date when="2021-05-25" />
+    </publicationStmt>
+    <sourceDesc>
+      <biblStruct>
+        <monogr>
+          <editor>Usener and Radermacher</editor>
+          <title xml:lang="lat">Dionysii Halicarnasei Opuscula 2.1</title>
+          <imprint>
+            <pubPlace>Leipzig</pubPlace>
+            <publisher>Teubner</publisher>
+            <date>1904</date>
+          </imprint>
+        </monogr>
+        <ref target="https://hdl.handle.net/2027/uc1.c020256950?urlappend=%3Bseq=155">HthiTrust</ref>
+      </biblStruct>
+    </sourceDesc>
+  </fileDesc>
+  <encodingDesc>
+    <editorialDecl>
+      <correction>
+        <p>Lace v. 0.6.18 copyright 2013-2020, Bruce Robertson, Dept. of Classics,
+        Mount Allison University. Developed through the support of the 
+        <ref target="https://nbif.ca/en">NBIF</ref>.</p>
+      </correction>
+    </editorialDecl>
+    <p>The following text is encoded in accordance with EpiDoc standards and with the
+    CTS/CITE Architecture</p>
+    <refsDecl n="CTS">
+      <cRefPattern n="book" matchPattern="(\w+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
```

Comment:
Is this correct? we have a cref pattern n="book" and a descriptor indicating "section" — your pattern needs to match the container description in the document